src/tests/src/libs/antares/test_utils.cpp

```diff
@@ -18,6 +18,13 @@ T beautify(const T& in) {
 constexpr auto beautifyStd = beautify<std::string>;
 constexpr auto beautifyYuni = beautify<Yuni::String>;
 
+std::string transformNameToId(const AnyString& name)
+{
+    std::string res;
+    Antares::TransformNameIntoID(name, res);
+    return res;
+}
+
 }
 
 BOOST_AUTO_TEST_CASE(test_beautify_name_std)
```

Comment:
About the 2 separate lines:
it's important to see that the size of those strings is 5, and not 4 as we could believe since there is indeed only 4 letters.

For comments about naming and signature, as discussed I agree, but this should be done in a separate PR!